src/dms-preview/azext_dms/_params.py

```diff
@@ -5,12 +5,22 @@
 
 from knack.arguments import CLIArgumentType
 
-from azure.cli.core.commands.parameters import tags_type
+from azure.cli.core.commands.parameters import resource_group_name_type, tags_type
 
 
 def load_arguments(self, _):
     name_arg_type = CLIArgumentType(options_list=['--name', '-n'], metavar='NAME')
 
+    with self.argument_context('dms task') as c:
+        c.argument('group_name', resource_group_name_type)
+        c.argument('service_name', options_list=['--service-name'], help="The name of the Service.")
```

Comment:
Is there a list with all allowed service name? Which kind of service name should be provided here?

Reply:
There is no list. The name of the user's service is user-defined. They would know what service(s) they own.